app/javascript/src/components/Dashboard/Contacts/SideMenu.jsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from "react";
+
+import { Search, Settings, Plus } from "neetoicons";
+import { Typography } from "neetoui";
+import { MenuBar } from "neetoui/layouts";
+
+const SideMenu = ({ showContactMenu }) => {
+  const [isSearchCollapsed, setIsSearchCollapsed] = useState(true);
+
+  return (
+    <div className="flex">
+      <MenuBar showMenu={showContactMenu} title="Contacts">
```

Comment:
Why do we need this `flex` here?

Reply:
Yes, It can be removed. There is no effect of ```flex```  here.
Thanks for pointing that out.